docs/samples/git-setup/clustertemplates/version_4.Y/sno-ran-du/sno-ran-du-v1.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: o2ims.provisioning.oran.org/v1alpha1
+kind: ClusterTemplate
+metadata:
+  name: sno-ran-du.v1
+  namespace: sno-ran-du-v4-16
+spec:
+  name: sno-ran-du
+  version: v1
+  templates:
+    hwTemplate: sno-ran-du-template-configmap-v1
```

Comment:
As mentioned above, the hwTemplate name should be independent from the cluster template naming.